extensions/default/src/findViewportsByPosition.ts

```diff
@@ -74,24 +74,19 @@ const findViewportsByPosition = (
   const viewportsByPosition = { ...syncState.viewportsByPosition };
   const initialInDisplay = [];
 
-  for (const viewport of viewports) {
+  viewports.forEach(viewport => {
     if (viewport.positionId) {
       const storedViewport = {
         ...viewport,
         viewportOptions: { ...viewport.viewportOptions },
       };
       viewportsByPosition[viewport.positionId] = storedViewport;
```

Comment:
I am not sure why we are no longer doing the deleting. Is there a reason?

Reply:
it is causing problems for the new "no viewportIndex world", maybe it was needed when we were messed up in the "viewport index world"